### PR TITLE
Fix: Use proper level name stored to dataLayer

### PR DESCRIPTION
**Pull request by OndraM**
Once more  :smile: 

:facepalm:

### CODE REVIEW

File: src/LmcCookieConsentManager.js [hunk 1]
```diff
@@ -150,7 +150,7 @@ function pushToDataLayer(cookie) {
     event: 'CookieConsent-update',
     'CookieConsent.necessary': cookie.level.includes('necessary'),
     'CookieConsent.analytics': cookie.level.includes('analytics'),
-    'CookieConsent.ads': cookie.level.includes('ads'),
+    'CookieConsent.ad': cookie.level.includes('ad'),
```
Comment (literat) [inline src/LmcCookieConsentManager.js:153]: 🤦

**Review by literat — APPROVED** [review]

File: src/LmcCookieConsentManager.js (hunk 1 above)
Comment (OndraM) [reply src/LmcCookieConsentManager.js:153]: giphy

Comment (literat) [reply src/LmcCookieConsentManager.js:153]: In English, please ;-)

**Pull request merged**